packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -520,6 +524,44 @@ aab artifact already exists, continuing...''',
     createArtifactProgress.complete();
   }
 
+  Future<void> createWindowsReleaseArtifacts({
+    required String appId,
+    required int releaseId,
+    required String projectRoot,
+    required String releaseZipPath,
+  }) async {
+    final createArtifactProgress = logger.progress('Uploading artifacts');
+
+    try {
+      // logger.detail('Uploading artifact for $aabPath');
+      await codePushClient.createReleaseArtifact(
+        appId: appId,
+        releaseId: releaseId,
+        artifactPath: releaseZipPath,
+        arch: 'exe',
```

Comment:
Should this be a shared constant somewhere?

Reply:
Done